.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,89 @@
+name: Docker Image CI
+
+on:
+  push:
+    tags:
+      - v*
```

Comment:
should also include `master`

Reply:
Ok, to be clear: Every commit on `master` should push a new image with `latest` tag?

Reply:
correct